### ISSUE TITLE
Request for Evaluation on MFT25 Test Set

### ISSUE THREAD

**Issue by Xxiong01** [vranlee/SU-T#6]
First of all, thank you for releasing the MFT25 dataset. It is very helpful for our research on fish tracking.I have trained a model on MFT25 and would like to evaluate it on the test set. Could you please inform me of the submission procedure for the test results? Thank you!

**Comment by vranlee** [comment]
Hi thanks! We plan to collaborate using an online evaluation platform after the official release next year. You can try dividing the training set yourself to evaluate the effect first.

**Comment by shiyi-git** [comment]
When will the ground truth files for the test set be available? I’d like to evaluate the test set results now, but they aren’t labeled yet.

**Comment by vranlee** [comment]
Hi all, MFT25's Testset Eval has been released on [Codabench](https://www.codabench.org/competitions/15922/) now. Thank you for your attention!
